packages/core/src/internal/utils/css.ts

```diff
@@ -39,9 +39,15 @@ export function updateElementStyles(el: HTMLElement, ...styleTuples: [string, st
 }
 
 export function pxToRem(pxValue: number) {
-  const baseProp = window
-    .getComputedStyle(document.body, null)
-    .getPropertyValue('--cds-global-typography-base-font-size');
-  const baseFontSize = (16 * parseInt(baseProp !== '' ? baseProp : '100%')) / 100;
-  return `${pxValue / baseFontSize}rem`;
+  let base = parseInt(document.documentElement.getAttribute('cds-base-font') as any);
```

Comment:
Would this attribute really comeback as `any`? Seems like it would be either a `string | undefined | null`.